Bootstrap v4 - (Badges) Badge - Danger.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Badge - Danger",
+    "heading": "badge-danger",
+    "description": "Documentation and examples for badges, our small count and labeling component.",
+    "usage": "Badges scale to match the size of the immediate parent element by using relative font sizing and em units.",
+    "snippet": "<span class=\"badge badge-danger\">Danger<\/span>\r\n",
+    "url": "https:\/\/getbootstrap.com\/docs\/4.0\/components\/badge\/",
+    "type": "Badges",
+    "library": "Bootstrap v4",
+    "created": "0000-00-00 00:00:00",
```

Comment:
I see all the creation dates are blank, @mwweb is there a reason  for this? We use these two dates to do detection of the state of the snippets. We may have an error here... I will need to check. Could you do a database update to change this creation date to lets say "2017-11-00 00:00:00" it must have happened with the import with the spreadsheet...

Reply:
When I added the snippets to my JCB for doing this, I did it as an import. It looks like it didn't process the data. I just did an update of the date on all of those, and will be resubmitting soon.

Reply:
IF you can fix it easy.. then great, since I can fix it very fast with little bash snippet.